load-test-framework/run.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+import time
+
+BASE_EXEC_PATH = 'java -Xmx20g -jar target/flic.jar '
+
+
+def kafka_pub_load_test():
+  # Runs a test which publishes to Kafka
+  args = BASE_EXEC_PATH + \
+  '--num_messages 10000000 \
+  --message_size 10 \
+  --topics [YOUR TOPIC HERE] \
```

Comment:
Yeah, it is strange that the README talks about command-line arguments, but those are not present here in the actual startup script.